src/hive/syncthing/logic.clj

```diff
@@ -1,4 +1,6 @@
-(ns hive.syncthing.logic)
+(ns hive.syncthing.logic
+  (:require [hive.utils :as utils]
+            [xml-in.core :as xml]))
```

Comment:
Have you looked at clojure.data.xml ? Is a contrib libary that usually is preferred